homeassistant/components/cover/services.yaml

```
@@ -1,77 +1,87 @@
 # Describes the format for available cover services
 
 open_cover:
-  description: Open all or specified cover.
-  fields:
-    entity_id:
-      description: Name(s) of cover(s) to open.
-      example: "cover.living_room"
+  description: Open all or specified cover
+  target:
+    entity:
+      domain: cover
 
 close_cover:
-  description: Close all or specified cover.
-  fields:
-    entity_id:
-      description: Name(s) of cover(s) to close.
-      example: "cover.living_room"
+  description: Close all or specified cover
+  target:
+    entity:
+      domain: cover
 
 toggle:
-  description: Toggles a cover open/closed.
-  fields:
-    entity_id:
-      description: Name(s) of cover(s) to toggle.
-      example: "cover.garage_door"
+  description: Toggles a cover open/closed
+  target:
+    entity:
+      domain: cover
 
 set_cover_position:
-  description: Move to specific position all or specified cover.
+  description: Move to specific position all or specified cover
+  target:
+    entity:
+      domain: cover
   fields:
-    entity_id:
-      description: Name(s) of cover(s) to set cover position.
-      example: "cover.living_room"
     position:
-      description: Position of the cover (0 to 100).
+      name: Position
+      description: Position of the cover
+      required: true
       example: 30
+      selector:
+        number:
+          min: 0
+          max: 100
+          step: 1
+          unit_of_measurement: "%"
+          mode: slider
 
 stop_cover:
-  description: Stop all or specified cover.
-  fields:
-    entity_id:
-      description: Name(s) of cover(s) to stop.
-      example: "cover.living_room"
+  description: Stop all or specified cover
+  target:
+    entity:
+      domain: cover
 
 open_cover_tilt:
-  description: Open all or specified cover tilt.
-  fields:
-    entity_id:
-      description: Name(s) of cover(s) tilt to open.
-      example: "cover.living_room_blinds"
+  description: Open all or specified cover tilt
+  target:
+    entity:
+      domain: cover
 
 close_cover_tilt:
-  description: Close all or specified cover tilt.
-  fields:
-    entity_id:
-      description: Name(s) of cover(s) to close tilt.
-      example: "cover.living_room_blinds"
+  description: Close all or specified cover tilt
+  target:
+    entity:
+      domain: cover
 
 toggle_cover_tilt:
-  description: Toggles a cover tilt open/closed.
-  fields:
-    entity_id:
-      description: Name(s) of cover(s) to toggle tilt.
-      example: "cover.living_room_blinds"
+  description: Toggles a cover tilt open/closed
```

Maybe this should become
```suggestion
  description: Toggles a cover tilt open/closed
```